### PR TITLE
translations(readme): add turkish

### PR DESCRIPTION
**Pull request by berat**
### Description
I translated the readme document into Turkish and contributed. 

### Affected components

- [x] Docs

### Related Issues
Close to #3891

### CODE REVIEW

File: translations/README.turkish.md [hunk 1]
```diff
@@ -0,0 +1,219 @@
...
+## Katkıda bulunmak
+
+Daha fazla detay için [katkıda bulunma kuralını](CONTRIBUTING.md) kontrol edin.
+
+## Marka varlıkları
+
+Hasura marka varlıkları (logolar, Hasura maskotu, rozetler vb.) [assets/brand](assets/brand) klasöründedir. Bunları Uygulamanızda/websitenizde kullanmaktan çekinmeyin. Hasura kullanılarak oluşturduğunuz uygulamalarınıza "Powered by Hasura" işareti eklerseniz çok seviniriz. ❤️
+
+<div style="display: flex;">
+  <img src="assets/brand/powered_by_hasura_blue.svg" width="150px"/>
```
Comment (marionschleifer) [inline translations/README.turkish.md]: Logos are not showing here:

image

File: translations/README.turkish.md [hunk 2]
```diff
@@ -0,0 +1,219 @@
...
+<a href="https://hasura.io">
+  <img width="150px" src="https://graphql-engine-cdn.hasura.io/img/powered_by_hasura_white.svg" />
+</a>
+```
+
+## Lisans
+
+Çekirdek GraphQL Engine [Apache License 2.0](https://www.apache.org/licenses/LICENSE-2.0) (Apache-2.0) altında bulunur.
+
+**Diğer türm içerikler** ([`sunucu`](server), [`cli`](cli) ve
```
Comment (marionschleifer) [inline translations/README.turkish.md]: Links for `server`, `cli`, `console`, `docs` and `community` are not working.

File: translations/README.turkish.md [hunk 3]
```diff
@@ -0,0 +1,219 @@
...
+[`konsol`](console) dizinleri hariç) [MIT License](LICENSE-community) altındadır.
+Bu, [`dokuman`](docs) ve [`topluluk`](community) üzerindeki her şeyi içerir.
+directories.
+
+## Çeviriler
```
Comment (marionschleifer) [inline translations/README.turkish.md]: Most links here are not working. Please check them again.

File: translations/README.turkish.md [hunk 4]
```diff
@@ -0,0 +1,219 @@
...
+- [Boşnakça :bosnia_herzegovina:](translations/README.bosnian.md) (:pray: [@hajro92](https://github.com/hajro92))
+- [Rusca :ru:](translations/README.russian.md) (:pray: [@highflyer910](https://github.com/highflyer910))
+- [Yunanca 🇬🇷](translations/README.greek.md) (:pray: [@MIP2000](https://github.com/MIP2000))
+- [İspanyolca 🇲🇽](/translations/README.mx_spanish.md)(:pray: [@ferdox2](https://github.com/ferdox2))
+- [Endonezyaca :indonesia:](translations/README.indonesian.md) (:pray: [@anwari666](https://github.com/anwari666))
+- [Brezilya Portekizcesi :brazil:](translations/README.portuguese_br.md) (:pray: [@rubensmp](https://github.com/rubensmp))
+- [Almanca 🇩🇪](translations/README.german.md) (:pray: [@FynnGrandke](https://github.com/FynnGrandke))
+- [Çince :cn:](translations/README.chinese.md) (:pray: [@jagreetdg](https://github.com/jagreetdg) & [@johnbanq](https://github.com/johnbanq))
+
+Diğer dosyalar için çeviriler [burada](translations) bulunabilir.
```
Comment (marionschleifer) [inline translations/README.turkish.md]: This link is not working.

**Comment by berat** [comment]
I am slowly learning to support :) I corrected everything.

File: translations/README.md [hunk 5]
```diff
@@ -2,7 +2,7 @@
 
 | File            | Language      |
 |-----------------|---------------|
-| Readme          | [:uk: English](../README.md) ￨ [:fr: Français](README.french.md) ￨ [:jp: 日本語](README.japanese.md) ￨ [🇬🇷 Ελληνικά](README.greek.md) ￨ [🇲🇽 Spanish](README.mx_spanish.md) ￨ [:brazil: Português](README.portuguese_br.md) ￨ [🇩🇪 Deutsch](README.german.md) ￨ [:ru: Русский](README.russian.md) ￨ [:indonesia: Indonesian](README.indonesian.md) ￨ [:bosnia_herzegovina: Bosnian](README.bosnian.md) ￨ [:cn: 中文](README.chinese.md)  |
+| Readme          | [:uk: English](../README.md) ￨ [:fr: Français](README.french.md) ￨ [:jp: 日本語](README.japanese.md) ￨ [🇬🇷 Ελληνικά](README.greek.md) ￨ [🇲🇽 Spanish](README.mx_spanish.md) ￨ [:brazil: Português](README.portuguese_br.md) ￨ [🇩🇪 Deutsch](README.german.md) ￨ [:ru: Русский](README.russian.md) ￨ [:indonesia: Indonesian](README.indonesian.md) ￨ [:bosnia_herzegovina: Bosnian](README.bosnian.md) ￨ [:cn: 中文](README.chinese.md)  | [:tr: Türkçe](README.turkish.md) ￨
```
Comment (marionschleifer) [inline translations/README.md]: The Turkish one is not visible here:

image

File: translations/README.turkish.md [hunk 6]
```diff
@@ -0,0 +1,219 @@
...
+## İçerikler
+<!-- markdown-toc start - Don't edit this section. Run M-x markdown-toc-refresh-toc -->
+**İçerikler**
+
+- [Hızlı Başlangıç:](#hızlı-başlangıç)
+    - [Tek tıklamayla Heroku'ya deploy etme](#tek-tıklama-ile-herokuya-deploy-etme)
+    - [Diğer deploy etme yöntemleri](#diğer-deploy-etme-yöntemleri)
+- [Yapı](#yapı)
+- [İstemci tarafı takımlar](#i̇stemci-tarafı-takımlar)
+- [İş mantığı ekleme](#ş-mantığı-ekleme)
```
Comment (marionschleifer) [inline translations/README.turkish.md]: This link is not working.

File: translations/README.turkish.md [hunk 7]
```diff
@@ -0,0 +1,219 @@
...
+
+### Diğer deploy etme yöntemleri
+
+Docker tabanlı dağıtım ve gelişmiş yapılandırma seçenekleri için [deployment
+guides](https://docs.hasura.io/1.0/graphql/manual/getting-started/index.html) ya da 
+[install manifests](../install-manifests) bakınız.
+
+## Yapı
+
+Hasura GraphQL Engine, bir Postgres veritabnı örneğini önler ve istemci uygulamarınızdan HraphQL isteklerini kabul edebilir. Mevcut kimlik doğrulama sisteminizle çalışacak şekilde yapılandırılabilir ve kimlik doğrumaa sisteminizden dinamik değişkenlerle alan düzeyinde kurallar kullanarak erişim denetimini işleyebilir.
```
Comment (marionschleifer) [inline translations/README.turkish.md]:
```suggestion
Hasura GraphQL Engine, bir Postgres veritabnı örneğini önler ve istemci uygulamarınızdan GraphQL isteklerini kabul edebilir. Mevcut kimlik doğrulama sisteminizle çalışacak şekilde yapılandırılabilir ve kimlik doğrumaa sisteminizden dinamik değişkenlerle alan düzeyinde kurallar kullanarak erişim denetimini işleyebilir. 
```

File: translations/README.turkish.md [hunk 8]
```diff
@@ -0,0 +1,219 @@
...
+Çekirdek GraphQL Engine [Apache License 2.0](https://www.apache.org/licenses/LICENSE-2.0) (Apache-2.0) altında bulunur.
+
+**Diğer türm içerikler** ([`sunucu`](../server), [`cli`](../cli) ve
+[`konsol`](../console) dizinleri hariç) [MIT License](../LICENSE-community) altındadır.
+Bu, [`dokuman`](../docs) ve [`topluluk`](../community) üzerindeki her şeyi içerir.
+directories.
+
+## Çeviriler
+
+Bu Readme sayfası aşağıdaki çevirilerde mevcuttur:
```
Comment (marionschleifer) [inline translations/README.turkish.md]: Most of the translation links here are still not working. Could you have a look at them please?

**Comment by berat** [comment]
I hope this time everything is working. I checked it :) Plenty commitment was a pr :)

**Pull request merged**